Data/AppConfig/CMakeLists.txt

```diff
@@ -11,15 +11,17 @@ endforeach()
 # First generate then install it
 foreach(GEN_CONFIG_SRC ${GEN_CONFIG_SOURCES})
   # Get the filename only component
-  get_filename_component(CONFIG_NAME ${GEN_CONFIG_SRC} NAME_WE)
```

Comment:
Judging from https://cmake.org/cmake/help/latest/command/get_filename_component.html, using `NAME_WLE` here is a simpler fix that allows us to keep using `get_filename_component`. Haven't tried it out myself though :)

Reply:
You're right. That works great.